test/models/order_test.rb

```diff
@@ -1,7 +1,12 @@
-require 'test_helper'
+require "test_helper"
 
 class OrderTest < ActiveSupport::TestCase
   test "when Order exists" do
     assert Order
   end
+
+  test "response from bakery if order is done" do
+    orders_for_delivery = Order.completed_orders
```

Comment:
Order.completed_orders calls a class level method that gets /baked_jobs. It returns everything from that HTTParty request on the order model